rom/dev/src/flow/cold_reset/ldev_id.rs

```diff
@@ -35,79 +37,72 @@ rom_err_def! {
 #[derive(Default)]
 pub struct LocalDevIdLayer {}
 
-impl DiceLayer for LocalDevIdLayer {
+impl LocalDevIdLayer {
     /// Perform derivations for the DICE layer
     ///
     /// # Arguments
     ///
     /// * `env`   - ROM Environment
-    /// * `input` - Dice input
+    /// * `idevid_output` - Dice output
     ///
     /// # Returns
     ///
     /// * `DiceOutput` - key pair, subject identifier serial number, subject key identifier
-    fn derive(env: &RomEnv, input: &DiceInput) -> CaliptraResult<DiceOutput> {
+    pub fn derive(env: &RomEnv, idevid_output: &DiceOutput) -> CaliptraResult<DiceOutput> {
         cprintln!("[ldev] ++");
-        cprintln!("[ldev] CDI.KEYID = {}", input.cdi as u8);
-        cprintln!("[ldev] SUBJECT.KEYID = {}", input.subj_priv_key as u8);
+        cprintln!("[ldev] CDI.KEYID = {}", KEY_ID_CDI as u8);
+        cprintln!("[ldev] SUBJECT.KEYID = {}", KEY_ID_LDEVID_PRIV_KEY as u8);
         cprintln!(
             "[ldev] AUTHORITY.KEYID = {}",
-            input.auth_key_pair.priv_key as u8
+            idevid_output.subj_key_pair.priv_key as u8
         );
-        cprintln!("[ldev] FE.KEYID = {}", input.fe_key as u8);
-
-        // The UDS for this layer is  generated by previous layer
-        // (Initial Device ID DICE Layer).
-        //
-        // This is the decrypted Field Entropy
-        let uds = input.fe_key;
+        cprintln!("[ldev] FE.KEYID = {}", KEY_ID_FE as u8);
 
-        // Derive the DICE CDI from decrypted UDS
-        let cdi = Self::derive_cdi(env, uds, input.cdi)?;
+        // Derive the DICE CDI from decrypted field entropy
+        Self::derive_cdi(env, KEY_ID_FE, KEY_ID_CDI)?;
 
         // Derive DICE Key Pair from CDI
-        let key_pair = Self::derive_key_pair(env, cdi, input.subj_priv_key)?;
+        let ldevid_key_pair = Self::derive_key_pair(env, KEY_ID_CDI, KEY_ID_LDEVID_PRIV_KEY)?;
```

Comment:
Consider naming this to ldevid_key_pair.

Reply:
Done